app/src/main/java/ru/anisart/vv/MainActivity.kt

```diff
@@ -33,17 +33,17 @@ import java.util.*
 @RuntimePermissions
```

Comment:
**Reporter**: ANDROIDLINT
**Rule**: ClickableViewAccessibility
**Severity**: WARN
**File**: C:/Users/aanisimov/.jenkins/workspace/vv/app/src/main/java/ru/anisart/vv/MainActivity.kt L378

Accessibility in Custom Views&#10;Custom view &#96;&#96;WebView&#96;&#96; has &#96;setOnTouchListener&#96; called on it but does not override &#96;performClick&#96;&#10;If a &#96;View&#96; that overrides &#96;onTouchEvent&#96; or uses an &#96;OnTouchListener&#96; does not also implement &#96;performClick&#96; and call it when clicks are detected, the &#96;View&#96; may not handle accessibility actions properly. Logic handling the click actions should ideally be placed in &#96;View#performClick&#96; as some accessibility services invoke &#96;performClick&#96; when a click action should occur.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1605702852>*